package.json

```diff
@@ -38,7 +38,7 @@
   "files": [
     "dist/**/*"
   ],
-  "homepage": "https://github.com/MapColonies/ts-npm-package-boilerplate#readme",
+  "homepage": "https://github.com/MapColonies/export-interfaces#readme",
```

Comment:
update dependencies by  ts-server-boilerplate as they are newer

Reply:
what dependencies? as i know 'js-logger' etc.. should be update on the package user side